pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -566,6 +574,78 @@ public PartitionedTopicMetadata deserialize(String key, byte[] content) throws E
         return metadataFuture;
     }
 
+    protected static PartitionedTopicMetadata fetchPartitionedTopicMetadataCheckAllowAutoCreation(
+            PulsarService pulsar, String path, TopicName topicName) {
+        try {
+            return fetchPartitionedTopicMetadataCheckAllowAutoCreationAsync(pulsar, path, topicName)
+                    .get();
```

Comment:
If it's necessary to call Thread.sleep(), i think more suitable to put it here.

Reply:
This seems be a higher level above the zk path update. And maybe bring in no-need sleep().